career/building-confidence.md

```diff
@@ -44,7 +44,7 @@ You’ll communicate more effectively: Confidence allows you to speak concisely
 
 Slides
 
-- [Building Confidence](https://docs.google.com/presentation/d/1EauFdEp4UJcBK50ioj-giPwFeWf2FHc7i_rn0XGf5f8/edit)
```

Comment:
You had to make the changes in the existing presentation that was mentioned in the issue. i.e https://docs.google.com/presentation/d/1EauFdEp4UJcBK50ioj-giPwFeWf2FHc7i_rn0XGf5f8
But I don't see any changes in the above presentation.

Reply:
I thought we need to make a copy of the presentation and make changes in the copy and transfer ownership to techtonicaorg@gmail.com when I finish, according to https://github.com/Techtonica/curriculum#volunteering
So, @Kundan28 should I make changes in the current slide or follow this instruction?

Reply:
I'd prefer if it could stay on the same link.

If the content is exactly the same, but you've fixed the formatting & style, you can just copy and paste the updated one into the old slides.  Have you requested edit permissions?